src/assignment_list/assignmentlist.ts

```diff
@@ -293,6 +293,36 @@ class Assignment {
 
       }
     } else if (this.data.status == 'fetched') {
+        var refetchButton = document.createElement('button');
+        refetchButton.classList.add('btn', 'btn-danger', 'btn-xs');
```

Comment:
The class `btn-danger` does not seem to be used.

In this case I would call it `btn-warning`, since this is not supposed to overwrite anything.
And the style of this class could be defined in the *common.css* file, like `btn-default` and `btn-primary`.